src/components/menu-bar/user-avatar.jsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {thumbnailUrl} from '../../lib/user-thumbnail';
+
+import styles from './user-avatar.css';
+
+const UserAvatar = ({
+    className,
+    size,
+    src,
+    userId
+}) => (
+    <React.Fragment>
```

Comment:
This Fragment looks left-over from a previous iteration.

Reply:
You're right, it's not necessary now.